api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -253,26 +253,34 @@ public static CustomResourceDefinition kafka() {
         return client.customResources(CustomResourceDefinitionContext.fromCrd(kafka()), Kafka.class, KafkaList.class);
     }
 
+    public static MixedOperation<Kafka, KafkaList, Resource<Kafka>> kafkaOperation(KubernetesClient client, String version) {
+        return client.customResources(CustomResourceDefinitionContext.fromCrd(crd(Kafka.class, version)), Kafka.class, KafkaList.class);
+    }
+
     public static MixedOperation<Kafka, KafkaList, Resource<Kafka>> kafkaV1Alpha1Operation(KubernetesClient client) {
-        return client.customResources(CustomResourceDefinitionContext.fromCrd(crd(Kafka.class, Constants.V1ALPHA1)), Kafka.class, KafkaList.class);
+        return kafkaOperation(client, Constants.V1ALPHA1);
     }
 
     public static MixedOperation<Kafka, KafkaList, Resource<Kafka>> kafkaV1Beta1Operation(KubernetesClient client) {
-        return client.customResources(CustomResourceDefinitionContext.fromCrd(crd(Kafka.class, Constants.V1BETA1)), Kafka.class, KafkaList.class);
+        return kafkaOperation(client, Constants.V1BETA1);
     }
 
     public static MixedOperation<Kafka, KafkaList, Resource<Kafka>> kafkaV1Beta2Operation(KubernetesClient client) {
-        return client.customResources(CustomResourceDefinitionContext.fromCrd(crd(Kafka.class, Constants.V1BETA2)), Kafka.class, KafkaList.class);
+        return kafkaOperation(client, Constants.V1BETA2);
     }
 
     public static MixedOperation<Kafka, KafkaList, Resource<Kafka>> kafkaV1Operation(KubernetesClient client) {
-        return client.customResources(CustomResourceDefinitionContext.fromCrd(crd(Kafka.class, Constants.V1)), Kafka.class, KafkaList.class);
+        return kafkaOperation(client, Constants.V1);
     }
```

Comment:
Do we actually need this, given we don't currently have a `v1` for `Kafka`?

Reply:
This was there before, I just simplified the usage.